.clang-format

```diff
@@ -0,0 +1,104 @@
+# OSARA: Open Source Accessibility for the REAPER Application
+# .clang-format
+# Author: James Teh <jamie@jantrid.net>
+# Copyright 2023 Leonard de Ruijter
+# License: GNU General Public License version 2.0
+
+# Style based on Google's C++ style guide.
+BasedOnStyle: Google
+# Offset for access modifiers (public, protected, private).
+AccessModifierOffset: 0
+# Align code after an open bracket with block indentation.
+AlignAfterOpenBracket: BlockIndent
+# Do not align consecutive assignments.
+AlignConsecutiveAssignments: None
+# Do not align consecutive bit fields.
+AlignConsecutiveBitFields: None
+# Do not align consecutive declarations.
+AlignConsecutiveDeclarations: None
+# Do not align consecutive macros.
+AlignConsecutiveMacros: None
+# Do not align escaped newlines.
+AlignEscapedNewlines: DontAlign
+# Do not align operands in expressions.
+AlignOperands: DontAlign
+# Do not align trailing comments.
+AlignTrailingComments: Never
+# Do not allow short blocks (e.g., if statements without braces) on a single line.
+AllowShortBlocksOnASingleLine: Never
+# Do not allow short case labels in switch statements on a single line.
+AllowShortCaseLabelsOnASingleLine: false
+# Do not allow short enums on a single line.
+AllowShortEnumsOnASingleLine: false
+# Only allow empty functions on a single line.
+AllowShortFunctionsOnASingleLine: Empty
+# Do not allow short if statements on a single line.
+AllowShortIfStatementsOnASingleLine: Never
+# Do not allow short loops on a single line.
+AllowShortLoopsOnASingleLine: false
+# Do not pack function call arguments together.
+# A function call’s arguments will either be all on the same line or will have one line each.
+BinPackArguments: false
```

Comment:
I'm not sure about this one or BinPackParameters. I don't love splitting a function call into many lines when there are really short parameters. I also don't think this is something we do often in current OSARA code, so this one seems like it might cause a lot of churn.

Reply:
All right, I'll remove that one and we'll see what this does.